installation/local-hosting-vps/macos.md

```diff
@@ -4,8 +4,70 @@ description: Deploy Modmail on a MacOS machine.
 
 # MacOS
 
-TODO
-
 ## Prerequisites
 
-## Updating
+1. Minimum 2GB of RAM
+2. At least 2GB available disk space.
+3. Support MacOS version:
+   - MacOS 14 Sonoma (Tested)
+
+## Dependencies
+
+We will be using the following dependencies:
+
+* Python 3.10
+* Tools: `git`, `homebrew`
+* Additional Modmail requirements: `cairo`, `libxml2`, `libxslt`, `libffi`
+
+{% hint style="info" %}
+All code blocks should be executed in bash and line by line unless specified otherwise.
+{% endhint %}
+
+To install these dependencies, we will be using Terminal (zsh):
+
+```bash
+brew install python3 cairo libxml2 libxslt libffi
+pip3 install cairosvg
+```
+
+After that, install `pipenv` with:
+
+```bash
+python3.10 -m pip install pipenv
+```
+
+## Installing Bot
+
+Clone and change directory into the Modmail folder with:
+
+```bash
+git clone https://github.com/modmail-dev/modmail
+cd modmail
+```
+
+Inside the Modmail folder, Install `pipenv` and its Python packages with:
+
+<pre class="language-bash"><code class="lang-bash"><strong>python3.10 -m pipenv install --python 3.10
+</strong></code></pre>
+
+Create a file named `.env` with `nano` and paste all the environmental variables (secrets) needed to run the bot via right-clicking in the nano editor. Refer to the steps in the [parent Installation page](../#preparing-your-environmental-variables) to find where to obtain these.
```

Comment:
I know this is just copied over from the Ubuntu guide, but I'd argue it'd probably be better to tell them to use ``cp .env.example .env`` so the template shows up. I didn't realize it was written like this and understand why alot of people have things spelled wrong in their .env when selfhosting

Reply:
I'll be changing the ubuntu guide as well